src/tagging/components/tag.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+ import style from './style';
+
+class Tag extends React.Component {
```

Comment:
imho you dont need a class here, you can simply use a const function instead?

Reply:
Done